packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -9,7 +9,14 @@
 		</div>
 		<div v-if="showPopper" ref="content" :class="popperClass">
 			<slot :close="handleClickaway" name="content">
-				{{ content }}
+				<KtPopoverIconTextItem
+					v-for="(option, index) in options"
+					:key="index"
+					:icon="option.icon"
+					:isDisabled="option.isDisabled"
+					:label="option.label"
+					@click="clickHandler(option)"
```

Comment:
i noticed you can't call this `handleClick` b/c there is another `handleClick`
there's also a `handleClickaway`
I'd say stick to the naming convention of this file at least
which names all handlers to be prefixed with `handle`

so i propose u cann this `handleItemClick`

while you rename `handleClick` to `handleAnchorClick` maybe